k8s.gcr.io/k8s-staging-artifact-promoter/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- linusa
```

Comment:
:upside_down_face:

Reply:
I'm glad I'm not the only one who does this.

Reply:
In all seriousness, should I remove myself from this list? I don't really care either way (because my own PRs won't get approval anyway).

Reply:
No, you should be in the owners. You have the wrong username here though :)

Reply:
> because my own PRs won't get approval anyway

I'm confused... why won't your PRs get approval? :thinking: 

You can also `/approve` your own PR if you are an approver in the OWNERS file btw.

Reply:
@cblecker Ohhhhhhhh wow I just saw that. Thank you.

@nikhita I thought you couldn't self-approve your own PRs, even if you are an approver. That's been my experience in the Promoter repo. /shrug